src/OpenRoad.cc

```diff
@@ -311,10 +311,7 @@ void OpenRoad::readDef(const char* filename,
 {
   if (!floorplan_init && !incremental && !child && db_->getChip()
       && db_->getChip()->getBlock()) {
-    logger_->error(
-        ORD,
-        48,
-        "You can't load a new DEF file as the db is already populated.");
+    logger_->info(ORD, 48, "Loading an additional DEF.");
```

Comment:
might be nice to preserve the old behavior and have better control of when 3D is allowed.

Reply:
I'll tackle this when I create commands to setup 3d.